spec/models/conference_spec.rb

```diff
@@ -61,6 +61,34 @@
     end
   end
 
+  describe "#cfp_status" do
+    context "when cfp_end_at not available" do
+      it "is open" do
+        conference = build(:conference, cfp_end_at: nil)
+
+        expect(conference.cfp_status).to eq('open')
+      end
+    end
+
+    context "when cfp_end_at available" do
+      context 'when cfp_end_at is in future' do
+        it "is open" do
+          conference = build(:conference, cfp_end_at: Time.now + 1.day)
+
+          expect(conference.cfp_status).to eq('open')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.